src/strutil.c

```diff
@@ -63,9 +70,11 @@ int str_caseends(const char *str, const char *ends) {
 
```

Comment:
This is really knit-picking, but you can remove a blank line here as well so that there is a single blank line between the functions.

Reply:
> single blank line between the functions

Uhhhm, actually, I follow a different style in that I break groups of functions doing similar things with *two* blank lines (also if they are longer). In that respect it's a bit like breaking up text into paragraphs in ordinary writing. There is no clear rule other than "look and feel" and a sense of what belongs together...

Reply:
I am more worried about having wrongly replaced a strcmp() with str_eq() in the other files of this PR (b/c that might break things).

Reply:
> There is no clear rule other than "look and feel" and a sense of what belongs together...

And having said that... if you feel str_casematch() "belongs" to the other functions feel free to remove that line, just push onto this commit (I don't mind).